test/results/title.html

```diff
@@ -0,0 +1,30 @@
+
+<style>
+  html {
+    font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Helvetica,Arial,sans-serif;
+  }
+  body {
+    margin: 30px auto;
+    max-width: 800px;
+  }
+  pre {
+    background: black;
+    display: grid;
+    padding: 16px;
+  }
+  span > code {
+    background: black;
+    padding: 4px;
+  }
+  .highlighted, .word {
+    background-color: rgba(255, 255, 255, 0.25);
+  }
+</style>
+<h2>Title</h2>
+<p>title="app.js"</p>
+<div data-rehype-pretty-code-fragment="">
+  <div data-rehype-pretty-code-fragment="">
+    <div data-rehype-pretty-code-title="">app.js</div>
+    <pre><code data-language="js" data-theme="default"><span class="line"><span style="color: #FF7B72">const</span><span style="color: #C9D1D9"> </span><span style="color: #79C0FF">code</span><span style="color: #C9D1D9"> </span><span style="color: #FF7B72">=</span><span style="color: #C9D1D9"> </span><span style="color: #79C0FF">true</span><span style="color: #C9D1D9">;</span></span></code></pre>
+  </div>
+</div>
```

Comment:
Basically wraps it with a fragment and the title is its own `div` node. Do we need anything else?

Reply:
Than would be awesome in order to add icons via css!